app/service/parsing_svc.py

```diff
@@ -27,12 +27,26 @@ def __init__(self):
 
                 # save facts to DB
                 for match in matched_facts:
-                    if not any(f['property'] == match['fact'] and f['value'] == match['value'] and f['score'] <= 0 for f in
-                               operation['facts']):
-                        await data_svc.create_fact(
-                            source_id=op_source[0]['id'], link_id=x['link_id'], property=match['fact'],
-                            value=match['value'], set_id=match['set_id'], score=1
-                        )
+                    update_op = await data_svc.explode_operation(dict(id=operation['id']))
```

Comment:
Updating the operation here is essential.
Example: I have a link that prints the same value twice in a row and has a parser line. If I don't update the operation with every match, in cases like these I'd save two completely equal facts.